io/include/pcl/io/vlp_grabber.h

```diff
@@ -80,13 +80,34 @@ namespace pcl
       virtual std::string
       getName () const;
 
+      /** \brief Allows one to customize the colors used for each of the lasers. */
+      void
+      setLaserColorRGB (const pcl::RGB& color,
+                        uint8_t laserNumber);
```

Comment:
You normally wouldn't need to redefine this method,  but this inheritance scheme is completely broken.

Reply:
This declaration is correct, because the definition is different from base class. (laser number check)
```cpp
void
pcl::HDLGrabber::setLaserColorRGB (const pcl::RGB& color,
                                   uint8_t laserNumber)
{
  if (laserNumber >= HDL_MAX_NUM_LASERS)
    return;

  laser_rgb_mapping_[laserNumber] = color;
}
```
```cpp
void
pcl::VLPGrabber::setLaserColorRGB (const pcl::RGB& color,
                                   uint8_t laserNumber)
{
  if (laserNumber >= VLP_MAX_NUM_LASERS)
    return;

  laser_rgb_mapping_[laserNumber] = color;
}
```

Reply:
You're right :+1: